servers/rendering/renderer_rd/environment/gi.cpp

```diff
@@ -3856,12 +3456,12 @@ void GI::process_gi(Ref<RenderSceneBuffersRD> p_render_buffers, const RID *p_nor
 	push_constant.proj_info[2] = (1.0f - p_projections[0].columns[0][2]) / p_projections[0].columns[0][0];
 	push_constant.proj_info[3] = (1.0f + p_projections[0].columns[1][2]) / p_projections[0].columns[1][1];
 
-	bool use_sdfgi = p_render_buffers->has_custom_data(RB_SCOPE_SDFGI);
+	bool use_hddagi = p_render_buffers->has_custom_data(RB_SCOPE_HDDAGI);
```

Comment:
In the case of hddagi not being used, we know that hddagi is initialized as null.